test/input_files/v4.3/assembly_checker/failed/failed_ena_api_with_mapping/failed_ena_api_with_mapping.vcf

```diff
@@ -0,0 +1,9 @@
+##fileformat=VCFv4.1
+##FILTER=<ID=PASS,Description="All filters passed">
+##fileDate=20150218
+##reference=ftp://ftp.1000genomes.ebi.ac.uk//vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz
+##source=1000GenomesPhase3Pipeline
+##contig=<ID=AB325691,assembly=b37,length=249250621>
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
```

Comment:
Could you add a header entry explaining the reason for failure? If you look at the oldest failed test files you can see an example of how we did it.

Reply:
I have added `##CauseOfFailure` tag to indicate what is the VCF designed for in 0edf59b .